data/.gitignore

```diff
@@ -0,0 +1,2 @@
+#JSON
```

Comment:
Great job putting your .gitignore in here, but remember that when you push an empty directory to Github, Github doesn't actually take that folder. 

So when I pulled down, I had failing tests everywhere because the tests couldn't find a ``/../data/pokemon`` directory to put files into because I think you probably had an empty pokemon folder when you pushed (which is good that you were cleaning out all your data). :) 

In the future, if you want to push an otherwise empty folder up, you can go into that folder and put a blank file in called ``.gitkeep`` just so you can have a file that will allow the folder to get pushed up.